README.md

```diff
@@ -12,9 +12,16 @@ To run the Aerolyzer app locally:
 
 * ```cd Aerolyzer/```
 * ```python manage.py runserver```
-  
+
 * Open your browser to http://127.0.0.1:8000/app/
 
+<<<<<<< HEAD
```

Comment:
remove these git conflict comments i.e.  <<<<<<< HEAD and >>>>>>> 8bcb558c2c76cab7363834d64753975e922d55dc